### ISSUE TITLE
Ace work on the passwordFilter function

### ISSUE THREAD

**Issue by ZooeyMiller** [FAC-11/RRMBe#49]
:ok_hand: at first when I was looking at it and I thought "huh, I womder why they're using a `for loop` instead of an array method..? Then I see, you're using `break` to make your code more performant. This is **super** cool and I'm really happy to be seeing these kinds of performance considerations in your code. 
:tada: :heart: :fire:

(feel free to close straight away, just wanted to say this on record)

**Comment by absorpheus** [comment]
I also agree, I really like the way you created a separate array that stores the results and then breaks out the loop when the array gets populated to 5 elements.

👍
